src/js/components/campaignpage/CampaignPage.jsx

```diff
@@ -574,32 +578,29 @@ const AdvertCard = ({ ad }) => {
 		<div className="position-relative" style={{ minHeight: "100px", maxHeight: "750px" }}>
 			<div className="position-relative ad-card">
 				<img src="/img/LandingBackground/mobile.png" className="w-100 invisible"/>
-				<img src="/img/LandingBackground/mobile.png" className="position-absolute d-none d-md-block mobileframe-shadow" style={{left: "-7%", top:"-9%", width:"113%", zIndex:0}}/>
+				<img src="/img/LandingBackground/mobile.png" className="position-absolute d-none d-md-block mobileframe-shadow" style={{left: "50%", top:"50%", width:"80%", zIndex:0, transform: "translate(-50%, -50%)"}}/>
 				{/*<img src="/img/redcurve.svg" className="position-absolute tv-ad-player" style={{height: "80%"}} />*/}
-				<img src="/img/LandingBackground/mobile_frame.png" className="position-absolute d-none d-md-block" style={{left: "-7%", width:"113%", top:"-9%", zIndex:2, pointerEvents:"none"}}/>
+				<img src="/img/LandingBackground/mobile_frame.png" className="position-absolute d-none d-md-block" style={{left: "50%", width:"80%", top:"50%", zIndex:2, pointerEvents:"none", transform: "translate(-50%, -50%)"}}/>
 				<div className="position-absolute theunit">
 					<GoodLoopUnit vertId={ad.id} size={size} />
 				</div>
 			</div>
 			{Roles.isDev() ? <DevLink href={'https://portal.good-loop.com/#advert/' + escape(ad.id)} target="_portal">Portal Editor</DevLink> : null}
+			{/*<span className="position-absolute" style={{ left: "50%", top: "50%", transform: "translate(-50%, -50%)", zIndex: 0 }}>If you're seeing this, you likely have ad-blocker enabled. Please disable ad-blocker to see the demo!</span>*/}
 		</div>
 	);
 };
+
 const AdvertPreviewCard = ({ ad, handleClick, selected = false }) => {
 	let size = 'landscape';
 	// Show when the campaign ran
 	// Fallback to ad creation date
-	const durationText = ad.start || ad.end ? <span>
-		{ad.start ? <Misc.RoughDate date={ad.start} /> : ''}
-		{ad.start && ad.end ? ' - ' : null}
-		{ad.end ? <Misc.RoughDate date={ad.end} /> : ''}
-	</span> : <span>
-		<Misc.RoughDate date={ad.created} />
-	</span>;
+	const durationText = <Misc.DateDuration startDate={ad.start} endDate={ad.end}/>;
```

Comment:
(minor) Let's move `durationText` directly into the returned jsx.

Reply:
done